problems/problems.go

```diff
@@ -0,0 +1,11 @@
+package problems
+
+import (
+	"io/ioutil"
+	"os"
+)
+
+func GetSets() ([]byte, error) {
```

Comment:
I think that we should add interfaces to problems, results and standings to be possible mocking in unit test.